homepairs/homepairsUI/src/state/account/actions.tsx

```diff
@@ -57,12 +57,12 @@ const storeAccountData = async (accountJSON: any) => {
  * @param {AccountTypes} accountType
  * @param {NavigationRouteHandler} navigation -navigator passed from calling component */
 export function ChooseMainPage(accountType: AccountTypes, navigation: NavigationRouteHandler) {
-    if(accountType === AccountTypes.PropertyManager){
-      navigation.navigate(navigationPages.PropertiesScreen);  
+    if(accountType === AccountTypes.Tenant){
+      navigation.navigate(navigationPages.TenantProperty);
       return;
     }
-    navigation.navigate(navigationPages.TenantProperty);
-}
+    navigation.navigate(navigationPages.PropertiesScreen);  
+  }
```

Comment:
Was this change in if statement necessary? should property manager be the default case then?